roles/download/defaults/main.yml

```diff
@@ -117,6 +117,7 @@ weave_version: 2.8.1
 pod_infra_version: "3.7"
 
 cilium_version: "v1.12.1"
+calico_cli_version: "v0.12.4"
```

Comment:
I guess it should be `cilium_cli_version`.

Reply:
THX